Makefile

```diff
@@ -39,6 +40,10 @@ TOOLS_DIR := $(ROOT_DIR)/hack/tools
 TOOLS_BIN_DIR := $(TOOLS_DIR)/bin
 export PATH := $(abspath $(TOOLS_BIN_DIR)):$(PATH)
 
+GH_ORG_NAME ?= kubernetes-sigs
+GH_REPO_NAME ?= cluster-api-provider-vsphere
+GH_REPO ?= $(GH_ORG_NAME)/$(GH_REPO_NAME)
+
```

Comment:
nit: please remove this extra line.

Reply:
yeah I will update the PR which this change.